x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -171,6 +172,23 @@ export function RulesPage() {
         'data-test-subj': 'rulesTableCell-name',
         render: (name: string, rule: RuleTableItem) => <Name name={name} rule={rule} />,
       },
+      {
+        field: 'tags',
+        name: '',
+        sortable: false,
+        width: '50px',
+        'data-test-subj': 'rulesTableCell-tagsPopover',
+        render: (tags: string[], item: RuleTableItem) => {
+          return tags.length > 0
```

Comment:
We don't want to always show the tags badge even if with 0 tags?

Reply:
No. Current functionality (current Stack Management > Rules and connectors page) doesn't show a `0` tags badge in case we have no tags.